README.md

```diff
@@ -58,14 +58,20 @@ Install Ansible and `ansible.eda` collection:
 pip install ansible
 ansible-galaxy collection install ansible.eda
 ```
+### 3. Set up env variables
+  Set up the following env variable(s) when testing/running local dev env
 
-### 3. Services
+```shell
+export EDA_DEPLOYMENT_TYPE=local
```

Comment:
Why? I don't see this envvar used anywhere

Reply:
The default is set here: 
https://github.com/ansible/eda-server/blob/main/src/eda_server/config/__init__.py#L23

Reply:
I am going tp move this to a note and reference that this is only needed as a temporary setting.